CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,8 @@
+- [cosmos-hive-auth-provider] [HARDENING] Configurable Identity Manager endpoing (#146)
+- [cosmos-gui] [FEATURE] Use the ID registered at the Identity Manager as Cosmos user (#118)
+- [cosmos-auth] [HARDENING] Rename server.js as cosmos_auth.js (#139)
+- [cosmos-gui] [HARDENING] Rename app.js as cosmos_gui.js (#135)
+- [cosmos] [HARDENING] Add documentation about how to consume context data from Sinfonier (#145)
+- [cosmos-gui] [HARDENING] Configurable logging level support (#45)
+- [cosmos-gui] [FEATURE] Remove ssh provision for users (#152)
```

Comment:
The first 7 lines are about issues already fixed in the previous release. They must not be here.